runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -7397,27 +7202,6 @@ TR::Register *J9::Power::TreeEvaluator::VMnewEvaluator(TR::Node *node, TR::CodeG
    bool isConstantZeroLenArrayAlloc = isConstantLenArrayAlloc && (node->getFirstChild()->getInt() == 0);
    bool canOutlineNew = false;
```

Comment:
This variable is no longer needed as all its uses have now been removed, so its declaration here should be removed as well.